eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -146,7 +146,6 @@ stages:
               Location: ${{ coalesce(parameters.Location, cloud.value.Location) }}
               Cloud: ${{ cloud.key }}
 
-- ${{ if contains(variables['Build.DefinitionName'], 'tests-weekly')  }}:
   - template: /eng/pipelines/templates/stages/python-analyze-weekly.yml
```

Comment:
I suspect you might need to decrease the indent here.

Reply:
Thanks. Fixed indentation and tested on a few pipelines.